File: Linguagem-Java/resolucao/resolucao_retangulo.java
// import java.util.Locale;
import java.util.Scanner;

public class resolucao_retangulo {
  public static void main(String[] args) {
  Scanner sc = new Scanner(System.in);
    
    // declaracao das variaveis a sere utilizadas na resolucao //
    double base, altura, area, perimetro, diagonal;

    // input para coleta dos valores a serem alocados nas variaveis //
    System.out.print("Base do retangulo: ");
    base = sc.nextDouble();
    System.out.print("Altura do retangulo: ");
    altura = sc.nextDouble();

    area = base * altura;
    perimetro = 2 * (base + altura);
    diagonal = Math.sqrt(Math.pow(base, 2.0) + Math.pow(altura, 2.0));

    System.out.println("Area = " + String.format("%.4f", area));
    System.out.println("Perimetro = " + String.format("%.4f", perimetro));
    System.out.println("Diagonal = " + String.format("%.4f", diagonal));
    
  sc.close();
  }
}

File: Linguagem-C#/resolucao/resolucao_retangulo.cs
using System;
namespace resolucao_retangulo
{
  class Program
  {
    static void Main(string[] args)
    {
      CultureInfo CI = CultureInfo.InvariantCulture;

      // declaracao das variaveis a serem utilizadas na resolucao //
      double bs, alt, area, peri, diag;

      // imput dos dados a serem trabalhados //
      Console.Write("Base do retangulo: ");
      bs = double.Parse(Console.ReadLine(), CI);
      Console.Write("Altura do retangulo: ");
      alt = double.Parse(Console.ReadLine(), CI);

      // montagem dos calculos para obtencao das respostas solicitadas //
      area = bs * alt;
      peri = 2 * (bs * alt);
      diag = Math.Sqrt(Math.Pow(bs, 2.0) + Math.Pow(alt, 2.0));

      // output dos dados obtidos com os calculos acima //
      Console.WriteLine("Area = " + area.ToString("F4", CI));
      Console.WriteLine("Perimetro = " + peri.ToString("F4", CI));
      Console.WriteLine("Diagonal = " + diag.ToString("F4", CI));
    }
  }
}
